config/repositories.yaml

```diff
@@ -33,14 +33,14 @@ repositories:
     master: https://ghcr.io/hewlettpackard/lustre-csi-driver
     useRemoteK: true
     remoteReference:
-      build: v0.0.6
+      build: 0.0.7
```

Comment:
I think these need the `v`. @roehrich-hpe?

Reply:
Good catch.